UI/window-basic-settings.cpp

```diff
@@ -2054,6 +2090,10 @@ void OBSBasicSettings::LoadAdvancedSettings()
 			"RecRBPrefix");
 	const char *rbSuffix = config_get_string(main->Config(), "SimpleOutput",
 			"RecRBSuffix");
+	bool replayBuf = config_get_string(main->Config(), "AdvOut",
```

Comment:
Here's the problem, this should be a `config_get_bool`, whatever this is doing with a string is probably nasty.

Reply:
That's exactly right, I just pushed the fix